src/layouts/partials/HeaderPartial/HeaderPartial.jsx

```diff
@@ -6,19 +6,24 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
+import Immutable from "immutable"
 import { Button, TopButtons } from 'components'
 import { sidesPush } from 'redux/sides/actions'
 import { pendingTransactionsSelector } from '@chronobank/core/redux/mainWallet/selectors/tokens'
 import TxModel from '@chronobank/core/models/TxModel'
+import { DUCK_WATCHER } from '@chronobank/core/redux/watcher/actions'
 
 import NotificationContent, { NOTIFICATION_PANEL_KEY } from 'layouts/partials/NotificationContent/NotificationContent'
 import LocaleDropDown from 'layouts/partials/LocaleDropDown/LocaleDropDown'
 
 import './HeaderPartial.scss'
 
 function mapStateToProps (state) {
+  const { pendingTxs } = state.get(DUCK_WATCHER)
```

Comment:
Please use appropriate selector (if exists) instead of 'direct' access.
If there is no selector, please create new one somewhere in 'selectors/models.js'